lib/generators/sorcery/install_generator.rb

```diff
@@ -75,7 +75,7 @@ def copy_migration_files
 
       # Define the next_migration_number method (necessary for the migration_template method to work)
       def self.next_migration_number(dirname)
-        if ActiveRecord::Base.timestamped_migrations
+        if ActiveRecord.timestamped_migrations
```

Comment:
Is this backwards compatible with Rails 6? That would be my only concern merging this in.